_config.yaml

```diff
@@ -85,18 +72,3 @@ exclude:
   - README.md
   - LICENSE.md
   - CITATION.cff
-
```

Comment:
This should be kept though. I don't think the site will build properly without this.